qt/applications/workbench/workbench/plotting/figurewindow.py

```diff
@@ -35,6 +35,8 @@ def _validate_workspaces(names: List[str]) -> List[bool]:
     for name in names:
         result = False
         ws = ads.retrieve(name)
+        if type(ws) == WorkspaceGroup:
```

Comment:
Sorry this is a bit picky! Can you use `isinstance` instead?
https://www.flake8rules.com/rules/E721.html#:~:text=A%20object%20should%20be%20be,can%20handle%20subclasses%20as%20well.